app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use App\Http\Requests\UpdateUserRequest;
+use App\Repositories\RepositoryInterface\UserRepositoryInterface;
+
+class UserController extends Controller
+{
+    protected $user;
+
+    public function __construct(UserRepositoryInterface $user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $getUsers = $this->user->getAll();
+
+        return view('user.users', compact('getUsers'));
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $getUser = $this->user->find($id);
```

Comment:
```suggestion
        $user = $this->user->find($id);
```

Tương tự mấy cái dưới